core/mondoo-aws-security.mql.yaml

```diff
@@ -81,54 +82,69 @@ policies:
           - uid: mondoo-aws-security-iam-user-no-inline-policies-check
           - uid: mondoo-aws-security-iam-user-no-inline-policies-check-single-user
       - title: AWS Lambda Function
+        filters: asset.platform == "aws"
```

Comment:
```suggestion
```

Reply:
With the `variants:` checks we don't need `filters:` e.g. ` mondoo-aws-security-lambda-concurrency-check-single` would never fire with the group filter in place.